File: src/main/java/BuildTypes.java
import javax.xml.bind.annotation.XmlAttribute;
import java.util.ArrayList;
import java.util.List;

/**
 * BuildTypes
 */
public class BuildTypes {
  private Integer count = null;

  @XmlAttribute
  private String href = null;

  private String nextHref = null;

  private String prevHref = null;

  private List<BuildType> buildType = new ArrayList<BuildType>();

  public Integer getCount() {
    return count;
  }

  public void setCount(Integer count) {
    this.count = count;
  }

  public String getHref() {
    return href;
  }

  public void setHref(String href) {
    this.href = href;
  }

  public String getNextHref() {
    return nextHref;
  }

  public void setNextHref(String nextHref) {
    this.nextHref = nextHref;
  }

  public String getPrevHref() {
    return prevHref;
  }

  public void setPrevHref(String prevHref) {
    this.prevHref = prevHref;
  }

  public List<BuildType> getBuildType() {
    return buildType;
  }

  public void setBuildType(List<BuildType> buildType) {
    this.buildType = buildType;
  }
}

File: src/main/kotlin/BuildTypes.kt
import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlElement
import java.util.ArrayList

/**
 * BuildTypes
 */
@XmlAccessorType(XmlAccessType.FIELD)
class BuildTypes {
    @XmlAttribute
    private val count: Int? = null

    @XmlAttribute
    private val href: String? = null

    @XmlAttribute
    private val nextHref: String? = null

    @XmlAttribute
    private val prevHref: String? = null

    @XmlElement(name = "buildType")
    var buildType: List<BuildType> = ArrayList()
}

